src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -841,9 +841,9 @@ public ResponseEntity<Void> setLoadflowParameters(
     @GetMapping(value = "/studies/{studyUuid}/loadflow/parameters")
     @Operation(summary = "Get loadflow parameters on study")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The loadflow parameters")})
-    public ResponseEntity<LoadFlowParametersValues> getLoadflowParameters(
+    public ResponseEntity<LoadFlowParametersInfos> getLoadflowParameters(
```

Comment:
Really need this DTO in the study service
I think we con remove it
See security analysis

Reply:
We need this mainly to prepare SecurityAnalysis and SensitivityAnalysis run parameters 
here : https://github.com/gridsuite/study-server/pull/500/files#diff-45de6aca36e4e99b6fb6535af0b109b4caad3c3801bf5cc1a0089cde075bb3b3R928